sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -328,9 +350,9 @@ CURLcode CurlSession::SendBuffer(uint8_t const* buffer, size_t bufferSize)
         }
         case CURLE_AGAIN:
         {
-          // start polling operation
+          // start polling operation with 1 min timeout
```

Comment:
Why did we choose 1 minute?

Reply:
as suggested in libcurl samples.
We don't have any analysis or anything about it.
We need a timeout to be set, and I don't think if it makes sense to expose that value to be configurable.
This just means that after one minute of waiting for a socket to be ready for usage, we would just give up and throw